dev/Breadcrumb/Breadcrumb.xaml

```diff
@@ -0,0 +1,66 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <!-- see https://docs.microsoft.com/en-us/windows/uwp/design/style/segoe-ui-symbol-font -->
+    <x:String x:Key="SplitButtonSecondaryButtonContent">&#xE76C;</x:String>
+    <x:String x:Key="SplitButtonSecondaryButtonDefaultContent">&#xE70D;</x:String>
+
+    <Style x:Key="BreadcrumbItemStyle" TargetType="local:BreadcrumbItem">
+
+        <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <Setter Property="FocusVisualMargin" Value="-3" />
+        <Setter Property="IsTabStop" Value="True"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:BreadcrumbItem">
+                    <Border
+                        Background="{TemplateBinding Background}"
+                        BorderBrush="{TemplateBinding BorderBrush}"
+                        BorderThickness="{TemplateBinding BorderThickness}">
+                        <local:SplitButton x:Name="SplitButton" Margin="2">
```

Comment:
> We discussed the use of a SplitButton and decided to use a Button instead in v1 (and a SplitButton possibly in a future release), with a caption (driven by the item template) and a chevron glyph.
> 
> Then the BreadcrumbItem can have a ChevronVisibility visual state group with states Collapsed and Visible.
> 
> ...and I suppose the ... Button can use the Button.Flyout property.

If you do move to support SplitButtons as a way to render child nodes for each Breadcrumb, the chevrons separating the node levels, need not be clickable right, so could be rendered as a TextBlock instead of a Button right?